renderers/config/interactive_config.go

```diff
@@ -6,6 +6,8 @@ import (
 	"time"
 )
 
+const defaultVisibleLines = 10
```

Comment:
Let's keep it 5 as it was before for backward compatibility?

```suggestion
const defaultVisibleLines = 5
```